.github/workflows/build-test-deploy.yml

```diff
@@ -122,11 +129,18 @@ jobs:
         uses: actions/cache/restore@v4
         with:
           path: |
-            build/
+            build
           key: build-${{ github.repository }}-${{ github.ref_name }}-${{ github.run_number }}
 
+      - name: Restore frontend build artefacts
+        uses: actions/cache/restore@v4
+        with:
+          path: |
+            src/main/app/dist
+          key: frontend-build-artefacts-${{ github.repository }}-${{ github.ref_name }}-${{ github.run_number }}
```

Comment:
Yes the run_number is the same for the entire workflow.

Reply:
But this new Gradle cache is not helping I'm afraid and that's because those Gradle Node tasks are very hard to cache: https://github.com/node-gradle/gradle-node-plugin/issues/81

Reply:
What I would like to solve is the issue that our frontend is being built I think 3 times in our GitHub workflow in different steps.. But so far no luck.

Reply:
Not such if this is possible at all. For another day. :-)